### PR TITLE
Remove deprecated at() matcher from user_ldap tests

### PR DESCRIPTION
**Pull request by come-nc**
Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

File: apps/user_ldap/tests/LDAPProviderTest.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ private function getServerMock(IUserLDAP $userBackend, IGroupLDAP $groupBackend)
 			 ->setMethods(['getUserManager', 'getBackends', 'getGroupManager'])
 			 ->setConstructorArgs(['', new \OC\Config(\OC::$configDir)])
 			 ->getMock();
-		$server->expects($this->at(1))
+		$server->expects($this->never())
```
Comment (come-nc) [inline apps/user_ldap/tests/LDAPProviderTest.php:61]: @blizzz Is it expected that getBackends is actually never called on the mock?

**Comment by blizzz** [comment]
i did not really write unit tests with `at()`, did it?

File: apps/user_ldap/tests/LDAPProviderTest.php (hunk 1 above)
Comment (blizzz) [reply apps/user_ldap/tests/LDAPProviderTest.php:61]: At least the LDAP Provider calls it upon construction https://github.com/nextcloud/server/blob/master/apps/user_ldap/lib/LDAPProvider.php#L53-L79

Comment (come-nc) [reply apps/user_ldap/tests/LDAPProviderTest.php:61]: Yes but not on the server, on the user manager and the group manager. Sound like I can just remove this expects from $server?

Comment (blizzz) [reply apps/user_ldap/tests/LDAPProviderTest.php:61]: ah, yes, that looks bogus, remove it 🗑️

Comment (come-nc) [reply apps/user_ldap/tests/LDAPProviderTest.php]: Done :broom:

**Review by blizzz — APPROVED** [review]

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**